### PR TITLE
add AVM FRITZBox-6591 / 6660 / 6670 / 6690

### PR DESCRIPTION
**Pull request by billy995**
add FRITZBox-6591-cable
add FRITZBox-6660-cable
add FRITZBox-6670-cable
add FRITZBox-6690-cable

### CODE REVIEW

File: elevation-images/AVM/FRITZBox-6670-CABLE-DE.front.png
Comment (danner26) [inline elevation-images/AVM/FRITZBox-6670-CABLE-DE.front.png:1]: Please remove all of the extra white space

File: elevation-images/AVM/FRITZBox-6670-CABLE-DE.rear.png
Comment (danner26) [inline elevation-images/AVM/FRITZBox-6670-CABLE-DE.rear.png:1]: Please remove all of the extra white space

File: elevation-images/AVM/FRITZBox-6690-CABLE-DE.rear.png
Comment (danner26) [inline elevation-images/AVM/FRITZBox-6690-CABLE-DE.rear.png:1]: Please remove all of the extra white space

**Comment by harryajc** [comment]
No Response in 5 months

**Pull request closed without merging**